htdocs/projet/tasks/time.php

```diff
@@ -1058,7 +1058,7 @@
 		if ($search_note) $sql .= natural_search('t.note', $search_note);
 		if ($search_task_ref) $sql .= natural_search('pt.ref', $search_task_ref);
 		if ($search_task_label) $sql .= natural_search('pt.label', $search_task_label);
-		if ($search_user > 0) $sql .= natural_search('t.fk_user', $search_user);
+		if ($search_user > 0) $sql .= ' AND t.fk_user = ' . intval($search_user);
```

Comment:
Can you try instead 
natural_search('t.fk_user', $search_user, 2);

Reply:
Done